scripts/CEE/hive-data-collection/script.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Collect data from Hive clusters in order to troubleshoot/debug cluster provisioning delays.
+# Input:
+#  (Env) CLUSTERID
+#        Description: The Internal Cluster ID for which data is being collected
+
+set -e
+set -o nounset
+set -o pipefail
+
+start_job(){
+    echo "Job started at $(date +"%Y-%m-%d %T")"
+    echo ".................................."
+    echo
+}
+
+finish_job(){
+    echo
+    echo ".................................."
+    echo "Job finished at $(date +"%Y-%m-%d %T")"
+}
+
+HIVE_NAMESPACE="uhc-production-${CLUSTER_ID}"
```

Comment:
The HIVE_NAMESPACE may a bit confusing as we have a namespace for hive itself. Maybe use `CLUSTER_NAMESPACE`